modules/vmware-csi-driver-reqs.adoc

```diff
@@ -29,4 +29,9 @@ To install the vSphere CSI Driver Operator, the following requirements must be m
 * Virtual machines of hardware version 15 or later
 * No third-party vSphere CSI driver already installed in the cluster
 
-If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
\ No newline at end of file
+If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
+
+[NOTE]
+====
+The VMware vSphere CSI Driver Operator  is only supported on OpenShift when the cluster has been deployed with `platform: vsphere` in the install manifest.
```

Comment:
It renders fine in the preview, but there's an extra space after "Operator" in this note.

Reply:
Ack